src/test/modelconfig_test.cpp

```diff
@@ -221,6 +221,98 @@ TEST(ModelConfig, parseShapeParam) {
     EXPECT_EQ(status, ovms::StatusCode::SHAPE_WRONG_FORMAT);
 }
 
+TEST(ModelConfig, parseShapeDynamicParam) {
+    using namespace ovms;
+    ovms::ModelConfig config;
+    // Valid
+    std::string valid_str1 = "(64:128,128,256:512,300:301)";
+    std::string valid_str2 = "{\"input\": \"(1, 3:6, 3, 200:100000)\"}";
+    std::string valid_str3 = "{\"input\": \"auto\", \"extra_input\": \"(10:20)\"}";
```

Comment:
How we can denote "totally dynamic" shape in config?

Reply:
-1

Reply:
This is documented and tested here:
https://github.com/openvinotoolkit/model_server/pull/1061/files#diff-8329452f352c6f0bdbc6084a887400e379bd3f443c0dfcd5806d310708fe09b4R279